cmd/skaffold/app/cmd/run.go

```diff
@@ -36,13 +37,16 @@ func NewCmdRun() *cobra.Command {
 		WithExample("Build, test, deploy and tail the logs", "run --tail").
 		WithExample("Run with a given profile", "run -p <profile>").
 		WithCommonFlags().
+		WithFlags(func(f *pflag.FlagSet) {
+			f.StringSliceVarP(&opts.TargetImages, "build-image", "b", nil, "Choose which artifacts to build. Artifacts with image names that contain the expression will be built only. Default is to build sources for all artifacts")
```

Comment:
```suggestion
			f.StringSliceVarP(&opts.TargetImages, "build-image", "b", nil, "Only build artifacts with image names that contain the given substring. Default is to build sources for all artifacts")
```
nit: I know that you took the same text as the existing `build-image` flag on `build`. But the grammar seems off.

Reply:
If the flag is the same elsewhere then we should instead add it to the [`flagRegistry`](https://github.com/GoogleContainerTools/skaffold/blob/master/cmd/skaffold/app/cmd/flags.go#L53).